lib/rubocop/cop/rspec/file_path.rb

```diff
@@ -82,30 +82,39 @@ def on_top_level_example_group(node)
         private
 
         def ensure_correct_file_path(send_node, described_class, arguments)
-          glob = glob_for(described_class, arguments.first)
-          return if filename_ends_with?(glob)
-
-          add_offense(send_node, message: format(MSG, suffix: glob))
+          pattern = pattern_for(described_class, arguments.first)
+          return if filename_ends_with?(pattern)
+
+          # For the suffix shown in the offense message, modify the regular
+          # expression pattern to resemble a glob pattern for clearer error
+          # messages.
+          offense_suffix = pattern.gsub('.*', '*').gsub('[^/]*', '*')
+            .sub('\.rb', '.rb')
```

Comment:
How about if we substitute as little as possible, so very narrowly matching the regex-specific parts of the string? E.g.
```suggestion
          offense_suffix = pattern.gsub('.*', '*').sub('[^/]', '')
            .sub('\.', '.')
```
What do you think?

Reply:
Sure, it's changed now.